src/main/kotlin/step2/calculator/Operator.kt

```diff
@@ -0,0 +1,13 @@
+package step2.calculator
+
+enum class Operator(private val symbol: String, val calculate: (Long, Long) -> Long) {
+    PLUS("+", { a, b -> Math.plus(a, b) }),
```

Comment:
```suggestion
    PLUS("+", { a, b -> a + b }),
```
이렇게하면 코드 양이 더 적을것 같습니다.
복잡도가 높지 않으니 추출하지 않아도 괜찮을 것 같아요.